setup.pp

```diff
@@ -96,13 +96,21 @@
   'pygame',
   'pyserial',
   'requests',
+  'cherrypy',
 ]
 
 python::pip { $python_packages:
   pip_provider => 'pip3',
   require      => Package[ $main_packages, $piweatherrock_packages, ],
 }
 
+# Run upgrade script to import current config values to new config file
+exec { 'import config':
+  path    => '/bin:/usr/bin',
+  command => "python3 /home/pi/PiWeatherRock/scripts/upgrade.py",
+  unless  => "grep "0.0.13" config.json",
```

Comment:
Line 111 causes a failure due to too many double quotes. In general, unless interpolation is happening, single quotes should be used in Puppet code.

```suggestion
  command => 'python3 /home/pi/PiWeatherRock/scripts/upgrade.py',
  unless  => 'grep "0.0.13" config.json',
```

Reply:
Applying the above makes this happen :) 

```
 pi@rock  ~/PiWeatherRock   metaMMA-new-config ●  sudo puppet apply /home/pi/PiWeatherRock/setup.pp
Notice: Compiled catalog for rock.home.technicalissues.us in environment production in 0.97 seconds
Notice: /Stage[main]/Main/Exec[import config]/returns: executed successfully
Notice: /Stage[main]/Main/Service[PiWeatherRock.service]: Triggered 'refresh' from 1 event
Notice: Applied catalog in 24.74 seconds
```

Reply:
Also, after running the upgrade, I have `"fullscreen": "True"` in my json config but nothing is displayed in the full screen dropdown box.

Reply:
> Line 111 causes a failure due to too many double quotes. In general, unless interpolation is happening, single quotes should be used in Puppet code. Also, if the config changes the service needs to be told to restart (notified)

Fixed the quotation issue and added the notify line :)